src/App.vue

```diff
@@ -297,6 +325,16 @@ export default {
     }
   },
 
+  watch: {
```

Comment:
The `watch` block is moved to before the lifecycle hooks. ESLint was complaining about that.
![watch should be above](https://github.com/bcgov/business-filings-ui/assets/122301442/6b236bf0-5e87-49f0-b49c-71f8a7e6364a)